backend/typescript/models/user.model.ts

```diff
@@ -26,7 +26,7 @@ const UserSchema: Schema = new Schema({
   },
   email: {
     type: String,
-    required: true,
+    required: false,
```

Comment:
I didn't change this value... I fetched and pulled from main first and then made my branch - I made sure my main was up to date as well so I'm unsure why this is here...